### ISSUE TITLE
TECH生成的三维人体模型看介绍视频存在弯腿的情况

### ISSUE THREAD

**Issue by 11898d** [huangyangyi/TeCH#40]
TECH生成的三维人体模型看介绍视频还是存在弯腿的情况，是否可以使用解决ECON项目中解决弯腿的方法同时去解决TECH中的生成三维人体模型弯腿的情况呢。来自于：
[Sapiens](https://rawalkhirodkar.github.io/sapiens/) 解决了弯曲腿的问题

**Comment by YuliangXiu** [comment]
Yes, you can refer to https://github.com/YuliangXiu/ECON/blob/master/docs/tricks.md#bending-legs to remove the bending legs
